### ISSUE TITLE
Introduction: f undefined

### ISSUE THREAD

**Issue by mbrookes** [zackchase/mxnet-the-straight-dope#329]
In the introduction: "Training Error: This is the error on the dataset used to find _f_ by minimizing the loss on the training set.", _f_ isn't defined- "What the f?" 😄 

In the following paragraph: "Test Error: This is the error incurred on an unseen test set. This can be off by quite a bit (statisticians call this overfitting). In real-life terms, this is the equivalent of screwing up the real exam despite doing well on the practice exams." would a better use of the analogy be that "this is the equivalent of focusing on studying the topics covered in the practice exams, so doing poorly in the real exam"?
